rubicon/objc/runtime.py

```diff
@@ -2097,14 +2105,21 @@ def __init__(self, func, restype=NOTHING, *arg_types):
 
         self.literal = BlockLiteral()
         self.literal.isa = addressof(_NSConcreteGlobalBlock)
-        self.literal.flags = BlockConsts.HAS_STRET | BlockConsts.HAS_SIGNATURE
+        self.literal.flags = BlockConsts.HAS_STRET | BlockConsts.HAS_SIGNATURE | BlockConsts.HAS_COPY_DISPOSE
         self.literal.reserved = 0
         self.cfunc = self.cfunc_type(self.wrapper)
         self.literal.invoke = cast(self.cfunc, c_void_p)
         self.descriptor = BlockDescriptor()
         self.descriptor.reserved = 0
         self.descriptor.size = sizeof(BlockLiteral)
 
+        self.copy_helper_type = CFUNCTYPE(c_void_p, c_void_p, c_void_p)
+        self.dispose_helper_type = CFUNCTYPE(c_void_p, c_void_p)
+        self.copy_helper = self.copy_helper_type(self.copyHelper)
+        self.dispose_helper = self.dispose_helper_type(self.disposeHelper)
```

Comment:
I'm not sure about the naming here - `self.copy_helper` and `self.copyHelper` are identical, except that one is snake_case and the other is dromedaryCase. I think it would be better to give them different names, such as `copy_helper_c` and `copy_helper_py` or something, that would make it less confusing to read.

(Same goes for `dispose_helper` and `disposeHelper`.)

Reply:
OK, will do.  Note: I'm a C++/ObjC guy so snakeCase is always nicer to me.. ha ha.  Will do.